libs/api/domains/work-machines/src/lib/models/getWorkMachines.ts

```diff
@@ -0,0 +1,155 @@
+import { Field, ID, ObjectType, registerEnumType } from '@nestjs/graphql'
+import { Action, ExternalLink } from '../api-domains-work-machines.types'
+
+registerEnumType(Action, { name: 'WorkMachinesAction' })
+registerEnumType(ExternalLink, {
+  name: 'WorkMachinesExternalLink',
+})
+
+@ObjectType('WorkMachinesWorkMachineCollection')
+export class WorkMachineCollection {
+  @Field(() => [WorkMachine], { nullable: true })
+  value?: Array<WorkMachine> | null
+
+  @Field(() => [CollectionLink], { nullable: true })
+  links?: Array<CollectionLink> | null
+
+  @Field(() => [Label], { nullable: true })
+  labels?: Array<Label> | null
+
+  @Field(() => Pagination, { nullable: true })
+  pagination?: Pagination | null
+}
+
+@ObjectType('WorkMachinesWorkMachine')
```

Comment:
We have an [exception in the naming convention](https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions#exception-to-the-solution) that the domain name itself can be used in singular and plural for queries, mutations, input and object types. Should be able to simplify most "WorkMachineWorkMachine" duplication in the schema.

Reply:
sweet